### PR TITLE
Centre comments r1

### PR DESCRIPTION
**Pull request by zackarno**
In this PR I've address the internal (CHD) feedback we received.

So as we found out `tmap_save()` creates `.png` files that look different when run on a mac (my local) vs ubuntu (GHA runner). This makes it very hard to iterate small aesthetic tweaks.

The latest email you received (from today) is the updated version. It was getting unimaginably time-consuming to tweak on both the mac and runner so i simplified it, but think i was still able to incorporate the comments. 

Since, I'm not sure the saga is over, I've left som "#commented" code in the tmap() process just as a reminder of what was already tried/didn't work. I think if everything else is fine we can merge to main so the CRON job can kick in w/ the updated version and we can get any last feedback more easily.  Once  validated, I can clean up /remove the "#commented" code

If we are lucky this might be solve in tmap v4 which hopefully will be stable soon. I'd also be curious if we see any issue in a ggplot produced `.png`. One way to test it would be to to add a line-break "\n" to a legend item. To me this was the most quickly obvious and recognizable difference.

### CODE REVIEW

File: R/email_funcs.R
Comment (caldwellst) [inline R/email_funcs.R:313]: Aren't you still using `tmap` here?

**Review by caldwellst — CHANGES_REQUESTED** [review]
So, one of overall question, I'm a bit confused as you removed the installation line for `tmap` but you are still using the `tm_...()` functions later on, just not `add_tmap()`. Do those functions come from somewhere else?

Overall, I think looking good, but I notice the font doesn't seem to be Source Sans Pro, so maybe trying to use that would be good. As well, the UN caveat about boundaries, maybe you could put some new lines in there so it fits over the ocean and could be in light grey or white? Or maybe pushing it slightly lower so it doesn't overlap with the country.

I think the fonts still seem slightly small, now the basins for instance could probably be a bit larger?

Last, shall we send back for review and only merge once we have the approval?

**Comment by zackarno** [comment]
> So, one of overall question, I'm a bit confused as you removed the installation line for `tmap` but you are still using the `tm_...()` functions later on, just not `add_tmap()`. Do those functions come from somewhere else?

`{tmap}` is still being installed in `.github/depends.R` . I had just removed the installation of tmap v4 which i tried for a moment and then realized it was too much work `add_tmap()` is from `email_utils.R`

> Overall, I think looking good, but I notice the font doesn't seem to be Source Sans Pro, so maybe trying to use that would be good. As well, the UN caveat about boundaries, maybe you could put some new lines in there so it fits over the ocean and could be in light grey or white? Or maybe pushing it slightly lower so it doesn't overlap with the country.

 i think black for the disclaimer would have  more contrast/readability than white or blue.  would have liked line breaks, but they come out too big from linux (not mac). I ended up just adding more space at the bottom and i think it's fine - it's small but it's a disclaimer!

> I think the fonts still seem slightly small, now the basins for instance could probably be a bit larger?
made legend + basin labels larger
> Last, shall we send back for review and only merge once we have the approval?

sure

**Comment by caldwellst** [comment]
Lots of force pushing ha! I think the last changes for me are:

* Use `scale_y_continuous_hdx()` or `expand_limits(y = 0)` so that the plot starts from 0%.
* Adjust colors so I can tell difference between Upper and Lower Niger.
* Ensure Source Sans 3 is used (I said Pro before, same font, but they deprecated the Source Sans Pro name). It looks like `sans` is being used?
* Goddamn font size, the disclaimer is wild now ha.

**Comment by zackarno** [comment]
Feedback check list

from seth

- [x] Use scale_y_continuous_hdx() or expand_limits(y = 0) so that the plot starts from 0%.
- [x] Adjust colors on line plots - make more distinct
- [x] Ensure Source Sans 3 is used
- [x] increase size of basin labels
- [x] make disclaimer more readable/add space

From Erika (in addition to feedback in email/annotated screenshot)

- [x] add country label
- [x] change legend symbol border for basins to blue
- [ ] Make gauge status indicated by symbol shape rather than just color (nice to have - not needed)

**Comment by zackarno** [comment]
> * Use `scale_y_continuous_hdx()` or `expand_limits(y = 0)` so that the plot starts from 0%.

I think it is set with: `limits = c(0, ymax_lim)` ?  ymax_lim is set dynamically as it can go over 100%

**Comment by caldwellst** [comment]
Yeah, that's the equivalent if you're already calling `scale_y_continuous()`, where you can also use the `expand` argument. I was just saying that `gghdx::scale_y_continuous_hdx()` is designed to do that by default, setting `expand = c(0, 0)` so there is no padding either side of the observed data in the plot.

**Review by caldwellst — APPROVED** [review]

**Pull request merged**
